packages/expo-apple-authentication/src/AppleAuthentication.types.ts

```diff
@@ -192,38 +206,63 @@ export enum AppleAuthenticationCredentialState {
   TRANSFERRED = 3,
 }
 
+// @needsAudit
 /**
- * A value that indicates whether the user appears to be a real person. You get this in the
- * realUserStatus property of a `Credential` object. It can be used as one metric to help prevent
- * fraud.
+ * An enum whose values specify the system's best guess for how likely the current user is a real person.
  *
  * @see [Apple
  * Documentation](https://developer.apple.com/documentation/authenticationservices/asuserdetectionstatus)
  * for more details.
  */
 export enum AppleAuthenticationUserDetectionStatus {
+  /**
+   * The system does not support this determination and there is no data.
+   */
   UNSUPPORTED = 0,
+  /**
+   * The system has not determined whether the user might be a real person.
+   */
   UNKNOWN = 1,
+  /**
+   * The user appears to be a real person.
+   */
   LIKELY_REAL = 2,
 }
 
+// @needsAudit
 /**
- * Controls the predefined text shown on the authentication button.
+ * An enum whose values control which pre-defined text to use when rendering an [`AppleAuthenticationButton`](#appleauthenticationappleauthenticationbutton).
  */
 export enum AppleAuthenticationButtonType {
+  /**
+   * "Sign in with Apple"
+   */
   SIGN_IN = 0,
+  /**
+   * "Continue with Apple"
+   */
   CONTINUE = 1,
   /**
-   * Requires iOS 13.2 or later.
+   * "Sign up with Apple" *(requires iOS 13.2 or higher)*
    */
   SIGN_UP = 2,
 }
 
+// @needsAudit
 /**
- * Controls the predefined style of the authenticating button.
+ * An enum whose values control which pre-defined color scheme to use when rendering an [`AppleAuthenticationButton`](#appleauthenticationappleauthenticationbutton).
  */
 export enum AppleAuthenticationButtonStyle {
+  /**
+   * White button with black text
```

Comment:
in `AppleAuthenticationUserDetectionStatus` the comments all ended with a `.` - should we do that here too?